packages/fether-electron/babel.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
It seems to me that this file is not needed either, because the babel config is bundled inside electron-webpack. Can you try to run without this file?

Reply:
If I remove this file then when I run `yarn && yarn test` it can't interpret the import syntax

I've pushed some commits to a separate branch 'luke-312-taskbar-tests' here https://github.com/paritytech/fether/compare/luke-312-taskbar-tests?expand=1

<details><p><pre>

</pre></p></details>